src/CTA.WebForms2Blazor/Helpers/ComponentSyntaxHelper.cs

```diff
@@ -1,14 +1,16 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Linq;
-using Microsoft.CodeAnalysis;
+using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 namespace CTA.WebForms2Blazor.Helpers
 {
     public static class ComponentSyntaxHelper
```

Comment:
Just a nitpick a lot of the methods use the word Build in front of them which carries a meaning that somehow the method will actually 'build' a project somehow. I tihnk the better meaning here is probably construct and for BuildOn maybe AddTo.